source/Cosmos.Build.Builder/CosmosTask.cs

```diff
@@ -319,8 +319,8 @@ private void Restore(string project)
 
       Section("Publish Tools");
       Publish(Path.Combine(mSourcePath, "Cosmos.Build.MSBuild"), Path.Combine(xVsipDir, "MSBuild"));
-      Publish(Path.Combine(mSourcePath, "IL2CPU"), Path.Combine(xVsipDir, "IL2CPU"));
-      Publish(Path.Combine(mSourcePath, "XSharp.Compiler"), Path.Combine(xVsipDir, "XSharp"));
+      Publish(Path.Combine(mSourcePath, "../../IL2CPU"), Path.Combine(xVsipDir, "Cosmos.IL2CPU"));
+      Publish(Path.Combine(mSourcePath, "../../XSharp.Compiler"), Path.Combine(xVsipDir, "Cosmos.XSharp"));
```

Comment:
I've changed the .Compiler to XSC now, just gonna do the other changes and commit. I didn't notice the IL2CPU.dll when I did it, but I've just looked again and it was there :) reverting back, I did it the first time because it said the project IL2CPU wasn't in xVsipDir but I did see Cosmos.IL2CPU

I'll find the output from where it errored